sequencer/closingchecker.go

```diff
@@ -11,6 +11,11 @@ import (
 // shouldCloseSequenceInProgress checks if sequence should be closed or not
 // in case it's enough blocks since last GER update, long time since last batch and sequence is profitable
 func (s *Sequencer) shouldCloseSequenceInProgress(ctx context.Context) bool {
+	if s.sequenceInProgress.IsZkCountersReachedMax {
+		log.Info("current sequence should be closed because it has reached max zk counters")
+		s.sequenceInProgress.IsZkCountersReachedMax = false
+		return true
+	}
```

Comment:
Similar to previous coment we should never see any counte being exceeded, as before this happens the executor stops and returns an OOC. On the other hand it may be interesting to have some loic to close a batch if it's close to exceed some counters (example counter usage over 80%), but I wouldn't implement something like this yet